src/qt/sendcoinsdialog.cpp

```diff
@@ -260,7 +262,7 @@ void SendCoinsDialog::on_sendButton_clicked()
         strFunds = tr("using") + " <b>" + tr("anonymous funds") + "</b>";
         QString strNearestAmount(
             BitcoinUnits::formatWithUnit(
-                model->getOptionsModel()->getDisplayUnit(), 0.1 * COIN));
+                model->getOptionsModel()->getDisplayUnit(), vecPrivateSendDenominations.back()));
```

Comment:
yes, the actual text is `(privatesend requires this amount to be rounded up to the nearest 0.01000010 tDASH)` (0.1 was inaccurate btw, should say 0.10000100 tDASH instead to be precise).